extensions/amp-story/1.0/amp-story-interactive-binary-poll.css

```diff
@@ -52,12 +52,17 @@
 }
 
 .i-amphtml-story-interactive-option-text-container {
+  width: 100%;
```

Comment:
Here and below `!important` statements please :))

Reply:
Done :)

Reply:
Done :)